### ISSUE TITLE
[Outland][Event] Shartuul's Transporter

### ISSUE THREAD

**Issue by Crispybuilt** [dalaranwow/dalaran-wow#3228]
**Object ID:** TEMP Shartuul's Transporter Platform *300128*
**Item ID:** Crystalforged Darkrune *32602*

**Issue present:** After you obtain **Crystalforged Darkrune** and go to **Shartuul's Transporter Platform** to activate it, nothing happens.

**How it should work:** *After activating the portal, a massive event should start where you have to control a "demon pet". The event consists of three "phases" or battles, each of which places the player in control of a different demon. Each phase has a variety of objectives and mobs to battle, but generally, the idea is to defeat your enemies without letting your demon perish.*

**Helpful Links:**
*http://wotlk.openwow.com/item=32602*
*http://wowwiki.wikia.com/wiki/Shartuul's_Transporter_(event)*
*http://wotlk.openwow.com/object=300128*

**Videos how it should work:**
*https://www.youtube.com/watch?v=Cvmn5NIs4b0*
*https://www.youtube.com/watch?v=HBvX4xh7UXg*
*https://www.youtube.com/watch?v=W4OVY7lzPVs*
*https://www.youtube.com/watch?v=Jsi6lHxVjuM*
*https://www.youtube.com/watch?v=-lGedjtC6Rw*

I hope this gets scripted soon so we can see the glory of this wonderful TBC event in full action.

**Comment by Coolkira** [comment]
Yes its not working and i spend a lot of time to get rep with Ogrilla. Its very interesting and  hard event,please repair it.

**Comment by Rushor** [comment]
ids: https://github.com/kelno/windrunner/blob/8fe4ffaaa1e21a41521cef119094d5a5fac00fce/src/scripts/outland/shartuul_event.cpp

**Comment by Rushor** [comment]
Ref: https://github.com/TrinityCore/TrinityCore/issues/18655
